Sage/carbon
# fix(switch): fix styling defect caused by box-sizing border-box css - FE-5738

if border-box css is applied globally, it effects the styling of the swtich component. This fix address this bug by applying box-sixing: border-box to the switch-slider span element.

fixes FE-5738

### Proposed behaviour

<img width="591" alt="Screenshot 2023-04-03 at 16 46 20" src="https://user-images.githubusercontent.com/56251247/229560905-d501f30e-09cf-4d27-9a3c-c2483629d4b9.png">

### Current behaviour

![Screenshot 2023-04-03 at 16 45 31](https://user-images.githubusercontent.com/56251247/229560745-4588c91b-ddbe-49de-979c-5d3d8f964358.png)

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

Switch component should look and behave exactly as it did before this change. 
- Codesandbox that was created for the ticket - https://codesandbox.io/s/lucid-faraday-unr1n1?file=/src/App.js
- Codesandbox that uses the same code but built from this PR - https://codesandbox.io/s/carbon-quickstart-forked-dow72s
